pkg/services/accesscontrol/manager/manager_test.go

```diff
@@ -0,0 +1,117 @@
+package manager
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/grafana/pkg/api/routing"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/registry"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/accesscontrol/database"
+	actesting "github.com/grafana/grafana/pkg/services/accesscontrol/testing"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+func overrideAccessControlInRegistry(cfg *setting.Cfg) AccessControlService {
+	ac := AccessControlService{
+		Cfg:           cfg,
+		RouteRegister: routing.NewRouteRegister(),
+		Log:           log.New("accesscontrol-test"),
+		AccessControlStore: &database.AccessControlStore{
+			SQLStore: nil,
+		},
+	}
+
+	overrideServiceFunc := func(descriptor registry.Descriptor) (*registry.Descriptor, bool) {
+		if _, ok := descriptor.Instance.(*AccessControlService); ok {
+			return &registry.Descriptor{
+				Name:         "AccessControl",
+				Instance:     &ac,
+				InitPriority: descriptor.InitPriority,
+			}, true
+		}
+		return nil, false
+	}
+
+	registry.RegisterOverride(overrideServiceFunc)
+
+	return ac
+}
+
+func setupTestEnv(t testing.TB) *AccessControlService {
```

Comment:
I think these test utility functions should start with `t.Helper()`.
```suggestion
func setupTestEnv(t testing.TB) *AccessControlService {
	t.Helper()

```